Cannot build TS projects that use this library

When trying to build with `tsc`, you get several errors like so:
```javascript
error TS2307: Cannot find module '../node_modules/axios/index'
```

The generated `index.d.ts` file for example contains the following axios imports:
```javascript
loginWithEmail: (email: string, password: string) => Promise<import("../node_modules/axios/index").AxiosResponse<API.LoginResponse>>;
loginWithUsername: (username: string, password: string) => Promise<import("../node_modules/axios/index").AxiosResponse<API.LoginResponse>>;
```

This is one level off where `node_modules` actually is:
```
- /project_root
  |- /node_modules
    |- /tiktok-api
      |- /lib
         - index.d.ts
```

Therefore in `index.d.ts` the path `../node_modules/axios/index` resolves to `tiktok-api/node_modules/axios/index`, which is invalid.

This can be fixed by setting [`baseUrl`](https://www.typescriptlang.org/docs/handbook/module-resolution.html) to `./lib` in `tsconfig.json`

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "allowSyntheticDefaultImports": false,
+    "baseUrl": "./lib",
     "declaration": true,
     "experimentalDecorators": true,
     "lib": ["es6"],

